acknowledgements/ag-wg-active.html

```diff
@@ -48,15 +48,14 @@ <h2>Participants of the AG WG active in the development of this document:</h2>
 		<li>Andy Heath (Invited Expert)</li>
 		<li>Shawn Henry (W3C)</li>
 		<li>Thomas Hoffman (Educational Testing Service)</li>
-		<li>Sarah Horton (The Paciello Group, LLC)</li>
+		<li>Sarah Horton (Invited Expert)</li>
 		<li>Stefan Johansson (Invited Expert)</li>
 		<li>Marc Johlic (IBM Corporation)</li>
 		<li>Rick Johnson (VitalSource | Ingram Content Group)</li>
 		<li>Crystal Jones (Microsoft Corporation)</li>
 		<li>Andrew Kirkpatrick (Adobe)</li>
 		<li>John Kirkwood (Invited Expert)</li>
 		<li>Jason Kiss (Department of Internal Affairs, New Zealand Government)</li>
-		<li>Maureen Kraft (IBM Corporation)</li>
 		<li>JaEun Ku (University of Illinois at Urbana-Champaign)</li>
 		<li>Patrick Lauke (TetraLogical)</li>
```

Comment:
Minor pedantic thing, but: any chance it could be changed to Patrick H. Lauke (with the middle initial)?

Reply:
Yep.

Reply:
wunderbar